src/AppBanner/index.tsx

```diff
@@ -68,14 +80,10 @@ export const AppBanner: React.FC<Props> = ({
         });
     };
 
-    if (!showBanner || !header || !body || !cta || !imageUrl) {
-        return null;
-    }
-
-    if (!isImageUrlAllowed(imageUrl)) {
-        console.log(`Image URL ${imageUrl} is not allowed`);
-        return null;
-    }
+    // if (!isImageUrlAllowed(imageUrl)) {
+    //     console.log(`Image URL ${imageUrl} is not allowed`);
+    //     return null;
+    // }
```

Comment:
I think this commented check can be removed (now happens in the `canRender` function).